### PR TITLE
Openssl enablement

### PR DESCRIPTION
**Pull request by MikeFultonDev**

### CODE REVIEW

File: bin/applypatch [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
```
Comment (fanbo-meng) [inline bin/applypatch]: If these scripts are openssl specific can we put them all under another directory?

Comment (MikeFultonDev) [reply bin/applypatch]: I thought they might end up being useful for other projects but I do need to clean them up to make them general purpose.

Comment (MikeFultonDev) [reply bin/applypatch]: You are right though - for now let me move them out of the way.

Comment (MikeFultonDev) [reply bin/applypatch]: i moved the code and cleaned it up and generalized it. Please re-review @fanbo-meng

Comment (MikeFultonDev) [reply bin/applypatch]: i am marking this as resolved since @fanbo-meng is out. @fanbo-meng please open an issue if you still see problems.

**Pull request merged**